charts/keycloak/values.yaml

```diff
@@ -12,12 +12,44 @@ deployment:
       kcMetricsEnabled: "true"
       kcTransactionXaEnabled: "false"
       keycloakAdmin: admin
-      keycloakAdminPassword: "EXAMPLE_KC_PASSWORD8675309"
+      keycloakAdminPassword: "EXAMPLE_KC_PASS8675309"
       kcProxyHeaders: xforwarded
     image:
       repository: quay.io/keycloak/keycloak
       tag: 24.0.4
   replicas: 1
+
+# this will load the theme, edit for the current release number
+initContainers:
```

Comment:
This entire block should be within the deployment. If we are going the route of referencing the theme url based on the modernization-api repo, we can just make that a variable. Most things including the command should not need to change regularly. 

I see some other direct references to 7.4.4, maybe we can make some assumptions and pass in just the version we are using (if we were always updating according to latest deployment we could reference the app version in Chart.yaml).
Nonetheless, it can be 
theme_version = 7.4.4
then bash to theme_version_dashes = sed '...'